internal/driver/glfw/menu_bar.go

```diff
@@ -16,11 +16,41 @@ var _ fyne.Widget = (*MenuBar)(nil)
 // MenuBar is a widget for displaying a fyne.MainMenu in a bar.
 type MenuBar struct {
 	widget.Base
-	Items []fyne.CanvasObject
-
+	Items      []fyne.CanvasObject
+	index      int
```

Comment:
index is quite generic, not clear what this means

Reply:
index is the currently focused item on the menu bar. Using arrow keys, the focus is moved and index is updated. This is also used on toolbars and tabs.

Reply:
So maybe `activeIndex` or `currentMenu` naming then?

Reply:
Just changed it to currentItem